src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -191,10 +191,39 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
     }
 
     /// <summary>
-    /// The <see cref="IEstimator{ITransformer}"/> for detecting a signal change on an
-    /// <a href="https://en.wikipedia.org/wiki/Independent_and_identically_distributed_random_variables"> independent identically distributed (i.i.d.)</a> time series.
-    /// Detection is based on adaptive kernel density estimation and martingales.
+    /// The <see cref="IEstimator{TTransformer}"/> to detect a signal change on an
+    /// <a href="https://en.wikipedia.org/wiki/Independent_and_identically_distributed_random_variables"> independent identically distributed (i.i.d.)</a>
+    /// time series based on adaptive kernel density estimation and martingales.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// To create this estimator, use [DetectIidChangePoint](xref:Microsoft.ML.TimeSeriesCatalog.DetectIidChangePoint(Microsoft.ML.TransformsCatalog,System.String,System.String,System.Int32,System.Int32,Microsoft.ML.Transforms.TimeSeries.MartingaleType,System.Double)).
+    ///
+    /// ### Input and Output Columns
+    /// There is only input column and its type is <xref:System.Single>.
+    ///
+    /// | Output Column Name | Column Type | Description|
+    /// | -- | -- | -- |
+    /// | All input columns | Any | All input columns would pass by without being modified.
+    /// | `Prediction` | Known-sized vector of <xref:System.Single> | It contains alert level (non-zero value means a change point), score, p-value, and martingale value.
```

Comment:
Is `Prediction` a vector of `Single` with elements containing alert level (non-zero value means a change point), score, p-value, and martingale value?

i.e. rather than a known size, is it size 4? where:
0: alert level
1: score
2: p-value
3: martingale value

Reply:
Yep.